tests/integration/test_diff.sh

```diff
@@ -34,13 +35,14 @@ teardown () {
 # logic can be added to a "cleanup stack", by cascading function calls
 # within traps. See tests/integration/test_mount.sh for an example.
 trap teardown EXIT
+if [ -e ${GOMTREE} ]; then
```

Comment:
Probably better to indicate a skip here by returning 77.

Also, if you want these tests to run on the Atomic Host platforms until gomtree is pulled in automatically, you can simply add to the YAML:

```yaml
packages:
  - gomtree
```